subgraphs/compound-forks/protocols/iron-bank/src/constants.ts

```diff
@@ -1,11 +1,16 @@
 import { Address, dataSource, log, BigInt } from "@graphprotocol/graph-ts";
 import {
-  AVALANCHE_BLOCKS_PER_YEAR,
   ETHEREUM_BLOCKS_PER_YEAR,
   FANTOM_BLOCKS_PER_YEAR,
   Network,
+  SECONDS_PER_YEAR,
 } from "../../../src/constants";
 
+// See https://github.com/messari/subgraphs/issues/1746 for
+// reason why this is the casef
+const AVALANCHE_BLOCKS_PER_YEAR = SECONDS_PER_YEAR;
```

Comment:
I feel like `AVALANCHE_BLOCKS_PER_YEAR` is not the best name for this. We could just use `SECONDS_PER_YEAR` and not create a psuedonym. bc that is not actually blocks/year on avax/optimism

Reply:
Makes sense. Fixed